woocommerce/woocommerce-android
# Feature/small fixes store creation flow

### Description
Fixes a number of small issues around store creation: 
- Fix navigation issues when pressing “Skip” button from store profiler industries step. Currently, if user clicked on "Skip" it will be redirected to Country picker step. What we want when user click "Skip" is to navigate to the next natural step in the flow. In this case, that'd be profiler industries step -> profiler user commerce journey. 
- Adds tracking to country picker step
- Store default selected country when opening country picker step
- Disabled "Continue" button from domain picker screen while loading. This ensures that a domain is selected before moving forward to the next step. 

### Testing instructions
- Trigger store creation flow 
- Check Skip button for profiler steps works as described in previous point
- Check that "Continue" button is disabled while domains are loading
- When landing on Country picker step, click continue right away without selecting any country. Check that store summary displays the default country based on device locale
- Check event is tracked when landing on country picker: 
`Tracked: site_creation_step, Properties: {"step":"store_profiler_country","is_debug":true}`


**Note**: To trigger the store creation flow you'll need to compile vanillaDebug flavour, and if the Google account you've set on your device is not US located you'll also need to disable the feature flag IAP_FOR_STORE_CREATION for debug builds.
